src/lib/steps/uglify.ts

```diff
@@ -38,7 +38,9 @@ export async function minifyJsFile(inputPath: string): Promise<string> {
     throw result.error;
   }
 
-  await writeFile(outputPath, result.code);
-  await writeFile(sourcemapOut, result.map);
+  await Promise.all([
```

Comment:
Per my comment about slowing the disk down, I'd like to see any and all parallel disk writes turned into synchronous disk flushes.

Reply:
Do you have any article about this? I'd like to know more :)